spec.html

```diff
@@ -13615,7 +13615,7 @@ <h1>Applying the `**` Operator</h1>
         <li>If _base_ &lt; 0 and _base_ is finite and _exponent_ is finite and _exponent_ is not an integer, the result is *NaN*.</li>
       </ul>
       <emu-note>
-        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity* or *-Infinity* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
+        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity*, *-Infinity* or *NaN* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behaviour is preserved for compatibility reasons.</p>
```

Comment:
```suggestion
        <p>The result of _base_ `**` _exponent_ when _base_ is *1* or *-1* and _exponent_ is *+Infinity*, *-Infinity*, or *NaN* differs from IEEE 754-2008. The first edition of ECMAScript specified a result of *NaN* for this operation, whereas later versions of IEEE 754-2008 specified *1*. The historical ECMAScript behavior is preserved for compatibility reasons.</p>
```

Reply:
the comma in the suggestion is fine, but spec should use Oxford English, so `behaviour` should not be changed.

Reply:
ah, fair enough - there's 125 current uses of the "u" and 1 use without. I'll update the suggestion.